liam2/tests/functional/simulation.yml

```diff
@@ -2473,6 +2475,13 @@ entities:
                       m_id=-1,
                       hh_id=-1)
 
+            test_default_value:
+                - show("test default value")
+                - qshow(defaulted_variable)
+                - qshow(undefaulted_variable)
+                - assertTrue(all(defaulted_variable == 99))
+                - assertFalse(all(undefaulted_variable == 99))

```

Comment:
replace this by assertEqual(xxx, -1)? (not sure it works)
or at least assertTrue(all(undefaulted_variable == -1))